keylime/registrar_client.py

```diff
@@ -71,25 +81,8 @@ def getData(
             logger.critical("Error: unexpected http response body from Registrar Server: %s", response.status_code)
             return None
 
-        if "aik_tpm" not in response_body["results"]:
-            logger.critical("Error: did not receive AIK from Registrar Server.")
-            return None
-
-        if "regcount" not in response_body["results"]:
-            logger.critical("Error: did not receive regcount from Registrar Server.")
-            return None
-
-        if "ek_tpm" not in response_body["results"]:
-            logger.critical("Error: did not receive EK from Registrar Server.")
-            return None
-
-        if "ip" not in response_body["results"]:
-            logger.critical("Error: did not receive IP from Registrar Server.")
-            return None
-
-        if "port" not in response_body["results"]:
-            logger.critical("Error: did not receive port from Registrar Server.")
-            return None
+        if not check_mandatory_fields(response_body["results"]):
+            return
```

Comment:
Could you please return `None` to keep the consistency?

Reply:
Sure, just think of it is a bit redundant here.